code/modules/medical/surgery_procs.dm

```diff
@@ -369,6 +370,7 @@ var/global/list/chestitem_whitelist = list(/obj/item/gnomechompski, /obj/item/gn
 					patient.TakeDamage("head", damage_high, 0)
 					take_bleeding_damage(patient, surgeon, damage_high, surgery_bleed = 1)
 					CREATE_BLOOD_SPLOOSH(patient)
+					display_slipup_image(surgeon, patient.loc)
```

Comment:
This should all be split into a proc, this exact 3-5 phrases are repeated like 12 times

Reply:
I put it all into a "do_slipup" proc.